contracts/Vault.sol

```diff
@@ -204,6 +206,7 @@ contract Vault is Escapable, Owned {
     function authorizePayment(
         string _name,
         bytes32 _reference,
+        address _token,
```

Comment:
I think it is a better strategy to create a second function for non-baseToken transfers... but we need to hear from jordi... it might be a few days